### DISCUSSION TITLE
When to expect a new release?

### DISCUSSION THREAD

**Discussion by hansingt** [CZ-NIC/pyoidc#876] [General]
Hi folks!

First of all, thanks for the work you provide us here!

The latest release is from Jul 3, 2023 and I know, that not much has happend in this repository since then, but especially the bug from https://github.com/CZ-NIC/pyoidc/pull/871 is occurring frequently for me.

Thus, I'd kindly like to ask, when I could expect a new release from this lib? Is there any chance, to get a new release containing this fix soon?

Kind regards
hansingt

**Comment by tpazderka** [comment]
I will prepare a new release this week.

**Reply by hansingt** [reply]
Thank you very much! This is way faster, than I'd have thought :)
